use_case_examples/cifar/cifar_brevitas_finetuning/FromImageNetToCifar.ipynb

```diff
@@ -64,7 +65,10 @@
    "source": [
     "dataset_name = \"CIFAR_100\"\n",
     "\n",
-    "device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
+    "# To use the CUDA backend for the Concrete compiler, install the GPU-supported version.\n",
+    "# Refer to the official guide:\n",
+    "# https://github.com/zama-ai/concrete-ml/blob/main/docs/guides/using_gpu.md#usage-preqreuisites\n",
```

Comment:
we'll need to merge the other PR so that the docs are published, then link to the actual doc

Reply:
The PR is ready